catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -55,9 +55,13 @@
 
 public class Look extends Image {
 
-	@Retention(RetentionPolicy.SOURCE)
-	@IntDef({ROTATION_STYLE_LEFT_RIGHT_ONLY, ROTATION_STYLE_ALL_AROUND, ROTATION_STYLE_NONE})
-	public @interface RotationStyle {}
+	public float getDistanceToTouchPositionInUserInterfaceDimensions() {
```

Comment:
PLEASE CHECK: For what is this actually *really* needed?

Reply:
When I used Android Studio Refactoring tool, it just split the sorting of functions!
Sorry for that, I have edit it now by another commit.

Reply:
Ahh, what I meant was: For what is this method used, what functionality does it serve, how does it impact Catroid? I may be a kind-of trick question, but actually, I am not sure myself ;-)

Reply:
After investigation and reading about annotations and @IntDef, its alternative to use enumeration, since enumeration is bad for performance. 
this method is being used at 
[ L54 SetRotationStyleAction.java](https://github.com/Catrobat/Catroid/blob/fb5dbaa40516c24e16b31d639174bd74ea54f2fe/catroid/src/main/java/org/catrobat/catroid/content/actions/SetRotationStyleAction.java#L54)
[ L283 ActionFactory.java](https://github.com/Catrobat/Catroid/blob/fb5dbaa40516c24e16b31d639174bd74ea54f2fe/catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java#L283) 
[ L102 SetRotationStyleAction.java](https://github.com/Catrobat/Catroid/blob/fb5dbaa40516c24e16b31d639174bd74ea54f2fe/catroid/src/main/java/org/catrobat/catroid/content/bricks/SetRotationStyleBrick.java#L102)

Reply:
Nono, I meant: For what do we need getDistanceToTouchPositionInUserInterfaceDimensions?

Reply:
we are using it here [L921 FormulaElement](https://github.com/Abdelaty/Catroid/blob/fb5dbaa40516c24e16b31d639174bd74ea54f2fe/catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java#L921)
it's kinda confusing for me, but I think it's return the distance from base point to touch position. 
I think also the FormulaElement class need to get refactoring.

Reply:
Sure, but please compare it to the other formula elements --- this one does not seem to be used anywhere in the app, I mean it is not part of anything the user can use in a formula in the Catroid app. Sure, there are tests, there's the code, but it cannot be used by the end-user, unless I am mistaken. I wonder what happened here. Can you check via "blame" who was responsible for this? Maybe we can clear that up, and either add the feature, or remove the code. What do you think?

Reply:
Sure, I will check "blame", And will investigate about the point from those code lines.

Reply:
I checked blame, and found that @robertriedl created it. But I don't think that he remembers it's 4 years already 😄 
![image](https://user-images.githubusercontent.com/23483020/77016430-4b671a80-6980-11ea-9d23-12ba588990a1.png)
I traced it and now I'm completing a tracing report to post it here before we can do any action.
And I investigated and it doesn't seem to be used in any part of the app, but I can't figure out what is this written for. So I may need some time to discover that.